src/plugins/intel_cpu/src/node.h

```diff
@@ -774,13 +774,16 @@ class Node {
                                        NameFromType(getType()));
     }
 
-    DnnlScratchPadPtr getScratchPad() const {
-        return context->getScratchPad(curNumaNode);
+    MemoryPtr getScratchPadMem(const DnnlMemoryDescPtr& desc) {
+        if (!scratchpadMem || !scratchpadMem->getDesc().isCompatible(*desc)) {
+            scratchpadMem = context->getScratchPad(curNumaNode)->createScratchPadMem(desc);
+        }
+        return scratchpadMem;
     }
 
-    MemoryPtr getScratchPadMem(const DnnlMemoryDescPtr& desc) {
+    MemoryPtr getScratchPadMem(const CpuBlockedMemoryDescPtr& desc) {
         if (!scratchpadMem || !scratchpadMem->getDesc().isCompatible(*desc)) {
-            scratchpadMem = getScratchPad()->createScratchPadMem(desc);
+            scratchpadMem = context->getScratchPad(curNumaNode)->createScratchPadMem(desc);
```

Comment:
Please simply join these two methods into one taking the base memory desc interface `getScratchPadMem(const MemoryDescPtr& desc)`

Reply:
The best idea! Done, thanks a lot!